internal/util/fetch/fetch.go

```diff
@@ -101,7 +101,11 @@ func cloneAndCopy(r *git.RepoSpec, dest string) error {
 			r.Path, r.OrgRepo, r.Ref)
 	}
 
-	if err := UpsertKptfile(dest, r); err != nil {
+	if err := kptfileutil.UpdateKptfileWithoutOrigin(dest, r.AbsPath(), false); err != nil {
```

Comment:
Why is this a special method? Why can't it just be UpdateKptfile() and origin not present being one of the cases.